.github/workflows/stackhpc-container-image-build.yml

```diff
@@ -117,6 +131,44 @@ jobs:
           sudo docker image ls --filter "reference=ark.stackhpc.com/stackhpc-dev/*:$(cat ~/kolla_tag)" > overcloud-container-images
         if: github.event.inputs.overcloud == 'true'
 
+      - name: Generate the images to scan
+        run: |
+          sudo docker image ls --filter "reference=ark.stackhpc.com/stackhpc-dev/*:$(cat ~/kolla_tag)" --format "{{.Repository}}:{{.Tag}}" > images-to-scan.txt
```

Comment:
We actually need to do this after we build the seed images, i.e. after:
`kayobe seed container image build --push -e kolla_base_distro=${{ inputs.distro }}`

Reply:
The kolla_tag is different for the overcloud & seed images, it would need a separate step to scan those

Reply:
I think we can use one step for both at the bottom, no matter which side we have built?

Reply:
```
      - name: Get built seed container images
        run: |
          sudo docker image ls --filter "reference=ark.stackhpc.com/stackhpc-dev/*:$(cat ~/kolla_tag)" > seed-container-images
        if: github.event.inputs.seed == 'true'
```

This bit of code looks identical to 

```
      - name: Get built overcloud container images
        run: |
          sudo docker image ls --filter "reference=ark.stackhpc.com/stackhpc-dev/*:$(cat ~/kolla_tag)" > overcloud-container-images
```

there shouldn't be an difference in the kolla tag this bit of code should list all the images that where built in the CI run. 

```
      - name: Generate the images to scan
        run: |
          sudo docker image ls --filter "reference=ark.stackhpc.com/stackhpc-dev/*:$(cat ~/kolla_tag)" --format "{{.Repository}}:{{.Tag}}" > images-to-scan.txt
        if: ${{ (github.event.inputs.overcloud == 'true') && (github.event.inputs.scan == 'true') || (github.event.inputs.scan_html == 'true') }}
```

Reply:
Except that the ~/kolla_tag file changes between those two steps.

Reply:
I think this is fixed now, but would appreciate a re-review